lib/shelley/exe/local-cluster.hs

```diff
@@ -86,6 +86,8 @@ import Test.Integration.Faucet
     )
 
 import qualified Cardano.BM.Backend.EKGView as EKG
+import Cardano.Wallet.PipeliningStrategy
+    ( variablePipelining )
```

Comment:
This import should be moved to the first section along the other normal, not qualified ones.